src/index.css

```diff
@@ -0,0 +1,182 @@
+@import url('https://fonts.googleapis.com/css2?family=Righteous&family=Russo+One&display=swap');
```

Comment:
Good job adding navigation for the cars, however, they are very difficult to navigate (I have 4 cars):

### In the desktop view:

- [ ] The "Previous" button sometimes is unclickable, and sometimes if I click it it just disappears:

![image](https://github.com/MasumaJaffery/Car-Rent_App/assets/57421823/1cece905-44dc-4f3a-8141-6b2fd06b26e0)

![image](https://github.com/MasumaJaffery/Car-Rent_App/assets/57421823/64c63935-6220-41c7-bbae-88b58b7b5222)

Therefore, if I go to the end of the collection, I can never go back unless I reload the page.

- [ ] The "delete" section has no navigation therefore it is impossible to delete the cars that don't appear on the screen:

![image](https://github.com/MasumaJaffery/Car-Rent_App/assets/57421823/9858ab8e-5639-44b4-857d-1e7f7bc76786)

Now that you've added some sort of navigation for your cards in the desktop view, kindly fix this issues so it gets to be functional. 💪 

_note: this was tested multiple times on both Chrome and Firefox_